autopts/ptscontrol.py

```diff
@@ -830,7 +835,12 @@ def get_bluetooth_address(self):
         return address
 
     def connect_to_dongle(self):
-        device_to_connect = None
+        args = parse_args()
+        if args.dongle:
+            device_to_connect = args.dongle
+            log("connect dongle ", args.dongle)
+        else:
+            device_to_connect = None
         devices = self._pts.GetDeviceList()
         for device in devices:
```

Comment:
I think we will still override device_to_connect in this `for` loop, so your change will have no effect.